lib/rules/max-nesting-depth/README.md

```diff
@@ -96,7 +96,9 @@ a .foo__foo .bar .baz {}
 
 ## Optional secondary options
 
-### `ignore: ["at-rules-without-declaration-blocks"]`
+### `ignore: ["blockless-at-rules"]`
+
+***Note: This option was previously called `at-rules-without-declaration-blocks`. See [the release planning docs](http://stylelint.io/user-guide/release-planning/) for details.***
```

Comment:
Let's drop the "See [the release planning docs](http://stylelint.io/user-guide/release-planning/) for details." bit.

I made a mistake advocating that in another PR. The releasing planning doc is a call for community help, and no help is required with these option renames.